internal/services/controllers/v1/task/process_payload_wal.go

```diff
@@ -4,38 +4,14 @@ import (
 	"context"
 )
 
-func (tc *TasksControllerImpl) runProcessPayloadWAL(ctx context.Context) func() {
-	return func() {
-		tc.l.Debug().Msgf("processing payload WAL")
-
-		partitions := []int64{0, 1, 2, 3}
-
-		tc.processPayloadWALOperations.SetPartitions(partitions)
-
-		for _, partitionId := range partitions {
-			tc.processPayloadWALOperations.RunOrContinue(partitionId)
-		}
-	}
-}
-
-func (tc *TasksControllerImpl) processPayloadWAL(ctx context.Context, partitionNumber int64) (bool, error) {
-	return tc.repov1.Payloads().ProcessPayloadWAL(ctx, partitionNumber, tc.pubBuffer)
-}
-
 func (tc *TasksControllerImpl) runProcessPayloadExternalCutovers(ctx context.Context) func() {
 	return func() {
 		tc.l.Debug().Msgf("processing payload external cutovers")
 
-		partitions := []int64{0, 1, 2, 3}
-
-		tc.processPayloadExternalCutoversOperations.SetPartitions(partitions)
-
-		for _, partitionId := range partitions {
-			tc.processPayloadExternalCutoversOperations.RunOrContinue(partitionId)
-		}
+		tc.processPayloadExternalCutoversOperations.RunOrContinue(0)
```

Comment:
Don't think we need to use the `operations` here - we should just be able to call the function directly and use the gocron option `gocron.WithSingletonMode(gocron.LimitModeReschedule)`